integration-tests/tests/fixtures/console-warnings.fixture.ts

```diff
@@ -1,36 +1,23 @@
 import { test as base, ConsoleMessage, expect } from '@playwright/test';
 
-type ConsoleWarningsFixtures = {
-    consoleWarnings: string[];
-};
-
-export const test = base.extend<ConsoleWarningsFixtures>({
-    consoleWarnings: [
-        async ({ page, browserName }, use) => {
-            const warnings: string[] = [];
-            const handleConsole = (msg: ConsoleMessage) => {
-                if (msg.type() === 'warning' || msg.type() === 'error') {
-                    const messageText = msg.text();
-                    // Filter out known harmless warnings
-                    if (
-                        !messageText.includes(
-                            'Form submission canceled because the form is not connected',
-                        )
-                    ) {
-                        warnings.push(`[${msg.type()}] ${messageText}`);
-                    }
+export const test = base.extend({
+    page: async ({ page, browserName }, use) => {
+        const handleConsole = (msg: ConsoleMessage) => {
+            if (msg.type() === 'warning' || msg.type() === 'error') {
+                const messageText = msg.text();
+                // Filter out known harmless warnings
+                if (
+                    !messageText.includes(
+                        'Form submission canceled because the form is not connected',
+                    ) &&
+                    browserName === 'chromium'
+                ) {
+                    expect(false, `Unexpected console ${msg.type()}: ${messageText}`).toBe(true);
```

Comment:
idk if `throw` is better than this? but nbd either way

Reply:
I don't know either, I wanted to use something that's part of the framework, usually I'd expect there to be a sort of "fail" method but I couldn't find one. I'll leave it for now.